tests/sqllogictests/suites/base/03_common/03_0038_spill_aggregator.test

```diff
@@ -4,7 +4,7 @@ set max_threads = 8;
 
 onlyif mysql
 statement ok
-set spilling_bytes_threshold_per_proc = 1024 * 1024 * 1;
+set aggregate_spilling_bytes_threshold_per_proc = 1024 * 1024 * 1;
```

Comment:
There is no logic tests to check the sort spilling works or not, will add it in another PR?

Reply:
> will add it in another PR?

Yes. 

The sort spill processor is not added to the pipeline yet. It's tested by unit tests in this PR.